monai/transforms/intensity/array.py

```diff
@@ -227,13 +227,15 @@ class ShiftIntensity(Transform):
         offset: offset value to shift the intensity of image.
         safe: if `True`, then do safe dtype convert when intensity overflow. default to `False`.
             E.g., `[256, -12]` -> `[array(0), array(244)]`. If `True`, then `[256, -12]` -> `[array(255), array(0)]`.
+        channel_wise: if `True`, shift the intensity for each channel of image with random offset
     """
 
     backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
 
-    def __init__(self, offset: float, safe: bool = False) -> None:
+    def __init__(self, offset: float, safe: bool = False, channel_wise: bool = False) -> None:
```

Comment:
Thank you, please also update the corresponding APIs in https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/intensity/dictionary.py and add test cases to the existing unit tests such as https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_shift_intensity.py